style.css

```diff
@@ -0,0 +1,158 @@
+:root {
+  --banner-bg-color: rgba(255, 255, 255, 0.301);
+}
+@font-face {
+  font-family: "ultraregular";
+  src: url("/fonts/ultra.woff") format("woff2"), url("/fonts/ultra.woff") format("woff");
+  font-weight: normal;
+  font-style: normal;
+}
+/***nav-bar***/
+.nav {
+  box-shadow: var(--nav-bar-shadow);
+}
+.nav-heading {
+  text-decoration: none;
+  color: var(--primary-color);
+  font-family: "ultraregular", sans-serif;
```

Comment:
save font family & font-weight  as a variable as you are using it at multiple places,

Reply:
I think I have a variable for font-weight forgot to use that. I will create variable for the font family.